mod_ci/controllers.py

```diff
@@ -545,8 +545,7 @@ def start_ci():
         elif event == "pull_request":
             # If it's a PR, run the tests
             commit = ''
-            gh_commit = None
```

Comment:
I don't agree on removing this line. This could leave the gh_commit variable unitialized if it's none of the scenario's below.